sideboard/lib/_services.py

```diff
@@ -91,9 +94,9 @@ def get_services(self):
         """
         return self._services
 
-    def _register_websocket(self, url=None, **ws_kwargs):
+    def _register_websocket(self, url=None, connect_immediately=True, **ws_kwargs):
         if url not in self._websockets:
-            self._websockets[url] = WebSocket(url, **ws_kwargs)
+            self._websockets[url] = WebSocket(url, connect_immediately=connect_immediately, **ws_kwargs)
```

Comment:
MAGFest doesn't do any websocket stuff, but this makes the websocket code a little more flexible.